api/RTCIceCandidatePairStats.json

```diff
@@ -127,10 +127,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "11"
```

Comment:
This (`availableOutgoingBitrate`) was added here in https://github.com/WebKit/WebKit/commit/ed457da32eb3b03501a8741588c14245e4974f6d going back to before the initial WebRTC support in Safari, so this checks out.